src/domain/events/utils.ts

```diff
@@ -29,18 +40,34 @@ export const getEventFilterVariables = (
   include: ['keywords,location'],
   text: getTextFromDict(query, 'text', undefined),
   inLanguage: getTextFromDict(query, 'inLanguage', undefined),
+  start: getDateString(query.date),
   ...options,
 });
 
+export const getDateString = (date?: string | string[]): string | null => {
```

Comment:
What is the reason to allow string array as date type? I'm just curious

Reply:
Because the query object can contain string or string array and that is passed as a parameter. This function makes the required checks.

Reply:
I'm open to suggestions if you have some ideas how this query object to form values conversion should be done. Not sure at all if this is good solution